app/Http/Controllers/Api/GithubWebhookController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Services\GithubWebhookService;
+use Illuminate\Http\Request;
+
+class GithubWebhookController extends Controller
+{
+    protected $webhookService;
+
+    public function __construct()
+    {
+        $this->webhookService = new GithubWebhookService();
+    }
+
+    public function handle(Request $request): void
```

Comment:
```suggestion
    public function __invoke(Request $request): void
```
Maybe `__invoke` if u have only one method in controller.

Reply:
changed to __invoke